docs/_integration-with-existing-apps-kotlin.md

```diff
@@ -4,94 +4,81 @@ import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import con
 
 The keys to integrating React Native components into your Android application are to:
 
-1. Set up React Native dependencies and directory structure.
-2. Develop your React Native components in JavaScript.
-3. Add a `ReactRootView` to your Android app. This view will serve as the container for your React Native component.
-4. Start the React Native server and run your native application.
-5. Verify that the React Native aspect of your application works as expected.
+1. Set up the correct directory structure
+2. Install the necessary NPM dependencies
+3. Adding React Native to your Gradle configuration
+4. Writing the TypeScript code for your first React Native screen
+5. Integrate React Native with your Android code using a ReactActivity
+6. Testing your integration by running the bundler and seeing your app in action
+
+## Using the Community Template
+
+While you follow this guide, we suggest you to use the [React Native Community Template](https://github.com/react-native-community/template/) as reference. The template contains a **minimal Android app** and will help you understanding how to integrate React Native into an existing Android app.
 
 ## Prerequisites
 
 Follow the guide on [setting up your development environment](set-up-your-environment) and using [React Native without a framework](getting-started-without-a-framework) to configure your development environment for building React Native apps for Android.
+This guide also assumes you're familiar with the basics of Android development such as creating Activities and editing the `AndroidManifest.xml` file.
 
-### 1. Set up directory structure
+## 1. Set up directory structure
 
-To ensure a smooth experience, create a new folder for your integrated React Native project, then copy your existing Android project to an `/android` subfolder.
+To ensure a smooth experience, create a new folder for your integrated React Native project, then **copy your existing Android project** to the `/android` subfolder.
```

Comment:
```suggestion
To ensure a smooth experience, create a new folder for your integrated React Native project, then **move your existing Android project** to the `/android` subfolder.
```
otherwise people will have two copies of the Android source files! xD